tests/Microsoft.Bot.Builder.TestBot.Json/testbot.schema

```diff
@@ -7251,129 +7251,148 @@
-					"$ref": "#/definitions/Microsoft.IActivityTemplate"
-				},
-				"threshold": {
-					"$ref": "#/definitions/numberExpression",
-					"title": "Threshold",
-					"description": "Threshold score to filter results.",
-					"default": 0.3
-				},
-				"activeLearningCardTitle": {
-					"$ref": "#/definitions/stringExpression",
-					"title": "Active learning card title",
-					"description": "Title for active learning suggestions card.",
-					"default": "Did you mean:"
-				},
-				"cardNoMatchText": {
-					"$ref": "#/definitions/stringExpression",
-					"title": "Card no match text",
-					"description": "Text for no match option.",
-					"default": "None of the above."
-				},
-				"cardNoMatchResponse": {
-					"$kind": "Microsoft.IActivityTemplate",
-					"title": "Card no match response",
-					"description": "Custom response when no match option was selected.",
-					"default": "Thanks for the feedback.",
-					"$ref": "#/definitions/Microsoft.IActivityTemplate"
-				},
-				"strictFilters": {
-					"$ref": "#/definitions/arrayExpression",
-					"title": "Strict Filters",
-					"description": "Metadata filters to use when calling the QnA Maker KB.",
-					"items": {
-						"type": "object",
-						"title": "Metadata filter",
-						"description": "Metadata filter.",
-						"properties": {
-							"name": {
-								"type": "string",
-								"title": "Name",
-								"description": "Name of filter property.",
-								"maximum": 100
-							},
-							"value": {
-								"type": "string",
-								"title": "Value",
-								"description": "Value to filter on.",
-								"maximum": 100
-							}
-						}
-					}
-				},
-				"top": {
-					"$ref": "#/definitions/numberExpression",
-					"title": "Top",
-					"description": "The number of answers you want to retrieve.",
-					"default": 3
-				},
-				"isTest": {
-					"type": "boolean",
-					"title": "IsTest",
-					"description": "True, if pointing to Test environment, else false.",
-					"default": false
-				},
-				"rankerType": {
-					"$ref": "#/definitions/stringExpression",
-					"title": "Ranker Type",
-					"description": "Type of Ranker.",
-					"oneOf": [
-						{
-							"title": "Standard ranker",
-							"description": "Standard ranker types.",
-							"enum": [
-								"default",
-								"questionOnly",
-								"autoSuggestQuestion"
-							],
-							"default": "default"
-						},
-						{
-							"$ref": "#/definitions/equalsExpression"
-						}
-					]
-				},
-				"$kind": {
-					"title": "Kind of dialog object",
-					"description": "Defines the valid properties for the component you are configuring (from a dialog .schema file)",
-					"type": "string",
-					"pattern": "^[a-zA-Z][a-zA-Z0-9.]*$",
-					"const": "Microsoft.QnAMakerDialog"
-				},
-				"$designer": {
-					"title": "Designer information",
-					"type": "object",
-					"description": "Extra information for the Bot Framework Composer."
-				}
-			}
+            "properties": {
```

Comment:
looks like some tab / space issue before and after.

same effected for below tests.schema

Reply:
Implemented review changes

Reply:
this format is not fixed. please cross check